testData/solutions/win32.json5

```diff
@@ -6278,6 +6278,203 @@
+                                        "transform": {
+                                            "type": "fluid.transforms.valueMapper",
+                                            "defaultInputPath": "SystemSettings_Accessibility_ColorFiltering_FilterType.value",
+                                            "defaultOutputPath": "value",
+                                            "match": [
+                                                {
+                                                    "inputValue": 0,
+                                                    "outputValue": 1
+                                                },
+                                                {
+                                                    "intputValue": 1,
+                                                    "outputValue": 2
+                                                },
+                                                {
+                                                    "inputValue": 2,
+                                                    "outputValue": 3
+                                                },
+                                                {
+                                                    "inputValue": 3,
+                                                    "outputValue": 4
+                                                },
+                                                {
+                                                    "inputValue": 4,
+                                                    "outputValue": 5
+                                                },
+                                                {
+                                                    "inputValue": 5,
+                                                    "outputValue": 6
+                                                }
+                                            ],
+                                            "noMatch": {
+                                                "outputValue": 0
+                                            }
+                                        }
+                                    }
+                                }
+                            ],
+                            "noMatch": {
+                                "outputValue": 0
+                            }
+                        }
+                    },
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.colorFilters.ShortcutKeyEnable": "SystemSettings_Accessibility_ColorFiltering_IsShortcutKeyEnabled.value"
+                }
+            }
+        },
+        "update": [
+            "settings.configure"
+        ],
+        "configure": [
+            "settings.configure"
+        ],
+        "restore": [
+            "settings.configure"
+        ],
+        "isInstalled": [
+            {
+                "type": "gpii.deviceReporter.alwaysInstalled"
```

Comment:
Question, can we check on a registry key rather than assuming that it's always installed? IIRC, this was recently added to Windows 10 and we might get into troubles on previous (but still valid) versions of Windows.

Reply:
For this windows features we don't have any reliable entry "single point" for knowing if they are installed, I can bet that the better approach will be to query the settings handler and let it check if they are present in the system, if not, it should return E_NOTIMPL, or similar, it's a functionality that could be studied.